Extensions/FileSystem/filesystemtools.js

```diff
@@ -97,6 +97,17 @@ gdjs.fileSystem.getUserdataPath = function(runtimeScene) {
   }
 };
 
+/**
+ * Get the path to 'UserData' folder.
+ * @return {string} The path to user*s "home" folder
+ */
+gdjs.fileSystem.getUserHomePath = function() {
+  if(require !== undefined) {
+    return require("os").homedir();

```

Comment:
Does this return the same directory as `electron.remote.app.getPath('home')` ?
If so it would be more consistent with how the other special folders are retrieved.

Reply:
good point! didn't see that was a thing